src/utils/renderDate.ts

```diff
@@ -0,0 +1,10 @@
+const moment = require('moment');
+// dates are in number and string format.
+function dateFromNow(date: string | number): string {
+  if (date === null) return 'Invalid date';
```

Comment:
instead of string can you return undefined or null I guess that would be easy to handle error or edge cases

Reply:
Rather we can have a better message to show. `Invalid date` shows up in the UI for available tasks as they don't have startedOn, endsOn. Either we can hide showing due date and startedOn OR we can show a better message over it.
What do you think @ivinayakg ?